packages/ui/src/composites/wui-list-select/index.ts

```diff
@@ -0,0 +1,71 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { globalStyles } from '../../utils/ThemeUtil'
+import '../../components/wui-text'
+import '../wui-wallet-image'
+import '../wui-all-wallets-image'
+import styles from './styles'
+import type { IWalletImage } from '../../utils/TypesUtil'
+
+@customElement('wui-list-select')
+export class WuiListSelect extends LitElement {
+  public static styles = [globalStyles, styles]
+
+  // -- state & properties ------------------------------------------- //
+  @property({ type: Array }) public walletImages?: IWalletImage[] = []
+
+  @property() public imageSrc? = ''
+
+  @property() public name = ''
+
+  @property() public status?: 'installed' | 'qr code' | 'recent'
+
+  @property({ type: Boolean }) public showAllWallets = false
+
+  // -- render ------------------------------------------------------- //
+  public render() {
+    return html`
+      <button>
+        ${this.templateAllWallets()} ${this.templateWalletImage()}
+        <wui-text variant="paragraph-500" color="fg-100">${this.name}</wui-text>
+        ${this.templateStatus()}
+      </button>
+    `
+  }
+
+  private templateAllWallets() {
+    if (this.showAllWallets && this.walletImages) {
```

Comment:
We need to account for less that 4 walletImages being passed here. There is some logic in V2 to use images that were given and replace missing ones with wallet placeholder graphics.